pkgs/development/libraries/libpsl/default.nix

```diff
@@ -38,6 +39,7 @@ in stdenv.mkDerivation rec {
     python3
     libxslt
   ] ++ lib.optionals enableValgrindTests [
+    glibc.debug
```

Comment:
(Should this be in valgrind’s `propagatedBuildInputs`, or be left up to its users?)

Reply:
Good question...!
I'm not sure every use-case of `valgrind` (in and out of nixpkgs) would expect `glibc` debug infos to be available. Maybe. 🤷